gwt-i18n/src/main/java/org/gwtproject/i18n/client/Localizable.java

```diff
@@ -20,8 +20,6 @@
  * 
  * deprecated use {@link org.gwtproject.i18n.shared.Localizable} instead
  */
-// Temporarily remove deprecation to keep from breaking teams that don't allow
-// deprecated references.
-// @Deprecated
+@Deprecated
```

Comment:
Was this deprecated in GWT2? if so why not removing it at all.?!

Reply:
Confusingly, no - it was deprecated, but then un-deprecated. From the comment, I assume that someone has a tool somewhere in Google that gets you in trouble if you use something deprecated, but rather than fix that code, they just commented out the deprecation notice... and never fixed it. 

Now the new deprecation notice is in place, and we'll remove this interface in a later release.